src/ios/OSBarcodeScanner.m

```diff
@@ -52,10 +56,11 @@ - (void)scan:(CDVInvokedUrlCommand*)command
 -(void)notificationHandler:(NSNotification *)notice{
     NSString *str = [notice object];
     CDVPluginResult* pluginResult = nil;
-    if (str != nil && [str length] > 0 && !([str isEqual: @"User closed before getting a result"])) {
+    if (str != nil && [str length] > 0 && !([str isEqual: @"Scanning cancelled."])) {
```

Comment:
Instead of using the raw string, you could use the error message associated with `kScanningCancelled`.

Reply:
fixed, thanks